pj/src/views/contact/Step1.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+<div class="box is-primary is-outlined">
+  <div class="notification">
+      <p id="step">STEP 1</p>
+      <h5 class="title is-5 has-text-centered">お客様の情報を入力してください</h5>
+  </div>
+  <div class="container is-size-5">
+    <h5 class="subtitle is-5 is-primary has-text-info">- 性別 -</h5>
+    <input name='status' type='radio' id='male' value="男性" v-model="customer.gender">
+    <label for="male">男性</label>
+    <input name='status' type='radio' id='female' value="女性" v-model="customer.gender">
+    <label for="female">女性</label>
+    
+  <br><br>
+
+    <h5 class="subtitle is-5 has-text-info">- 生年月日 -</h5>
+    <select id="year" v-model="customer.birthyear">
+      <option v-for="y in years" :key="y" :value="y.year">{{ y.text }}</option>
+    </select>
+    <label for="year">年</label>
+    <select id="month" v-model="customer.birthmonth">
+      <option v-for="n in 12" :key="n">{{ n }}</option>
+    </select>
+    <label for="month">月</label>
+    <select id="day" v-model="customer.birthday">
+      <option v-for="n in days" :key="n">{{ n }}</option>
+    </select>
+    <label for="day">日</label>
+
+  </div>
+</div>
+
+<div class="container" align="center">
+  <button class="button is-primary is-size-5" @click="forward">次へ進む ></button>
+</div>
+
+</template>
+
+<script>
+export default {
+  name: 'Step1',
+  data () {
+    return {
+      customer: 
+        {gender: '', birthyear: '1992', birthmonth: '1', birthday: '1'}
+    }
+  },
+  computed: {
+    days: function() {
+      return new Date(this.customer.birthyear, this.customer.birthmonth, 0).getDate();
+    },
+    years: function(){
+      let years = [{year: '', text: ''}];
+      let numItems = 100;
+      const jpEra = [
+        {era: '令和', period: 2019},
+        {era: '平成', period: 1989},
+        {era: '昭和', period: 1989},
+        {era: '大正', period: 1989},
+        {era: '明治', period: 1868}
+      ];
+
+      let startYear = new Date().getFullYear() - numItems;
+      let jpyear = 0;
+      
+      for (let i =0; i < 100; i++) {
+        let year = startYear + i;
+        for (let j = 0; j < jpEra.length; j++) {
+          if (year >= jpEra[j].period) {
+            jpyear = jpEra[j].era + (year-jpEra[j].period+1)
+            break;
+          }
+        }
+        years.push({ year: year, text: `${year}年(${jpyear})`})
+      }
+      return years
+    }
+  },
+  methods: {
+    forward () {
+      this.$router.push({ name: 'Step2' })
+    },
+    greet: function (event) {
+        event.stopPropagation();
+        event.toggle('is-active')
+    }
+  }
```

Comment:
`()`の外側はスペースをあけません。また、`function`と記述するのかしないのか統一するようにしましょう。記述せずに省略するほうがいいかと思います。
```suggestion
    forward() {
      this.$router.push( { name: 'Step2' } )
    },
    greet(event) {
        event.stopPropagation();
        event.toggle('is-active')
    }
  }
```